cogito/putter.go

```diff
@@ -262,7 +262,11 @@ func checkGitRepoDir(dir, owner, repo string) error {
 	if err != nil {
 		return fmt.Errorf(".git/config: remote: %w", err)
 	}
-	left := []string{"github.com", owner, repo}
+	parsedUrl, err := safeUrlParse(domain)
+	if err != nil {
+		return err
+	}
+	left := []string{parsedUrl.Host, owner, repo}
```

Comment:
I do think that its extremely important to test and verify all 3 components.
Input from Concourse git resource can come from anywhere (gitlab, bitbucket, github enterprise, etc...) so just comparing with hardcoded `github.com` is wrong.